example/App.tsx

```diff
@@ -60,6 +60,10 @@ function App(): React.JSX.Element {
     setStyles();
   }, []);
 
+  // if (fileUploadStyles) {
+  //   STYLES.setFileUploadStyle(fileUploadStyles);
+  // }
+
```

Comment:
remove these comments from here, and note these example steps on notion for customizability testing.

Reply:
created a separate styles file